From petkan@nucleusys.com Wed Mar 15 06:29:32 2006
Date: Wed, 15 Mar 2006 16:29:38 +0200 (EET)
From: Petko Manolov <petkan@nucleusys.com>
To: Greg KH <greg@kroah.com>
Subject: USB: rtl8150 small fix
Message-ID: <Pine.LNX.4.62.0603151626570.2354@localhost.localdomain>

This one is about announcing the device registration after the last check 
has been made.

From: Petko Manolov <petkan@nucleusys.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/rtl8150.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/net/rtl8150.c
+++ gregkh-2.6/drivers/usb/net/rtl8150.c
@@ -880,7 +880,6 @@ static int rtl8150_probe(struct usb_inte
 	}
 	fill_skb_pool(dev);
 	set_ethernet_addr(dev);
-	info("%s: rtl8150 is detected", netdev->name);
 	
 	usb_set_intfdata(intf, dev);
 	SET_NETDEV_DEV(netdev, &intf->dev);
@@ -888,6 +887,9 @@ static int rtl8150_probe(struct usb_inte
 		err("couldn't register the device");
 		goto out2;
 	}
+
+	info("%s: rtl8150 is detected", netdev->name);
+
 	return 0;
 
 out2:
